src/SOS/Strike/strike.cpp

```diff
@@ -7587,6 +7584,38 @@ class CNotification : public IXCLRDataExceptionNotification5
      */
     STDMETHODIMP OnCodeGenerated(IXCLRDataMethodInstance* method)
     {
+#ifndef FEATURE_PAL
+        if (g_isDesktopRuntime)
+        {
+            // Some method has been generated, make a breakpoint and remove it.
+            ULONG32 len = mdNameLen;
```

Comment:
Maybe write why you need to break on generated code for non-Win platforms.

Reply:
added comment.